memgpt/cli/cli_config.py

```diff
@@ -116,24 +133,51 @@ def configure_model(config: MemGPTConfig, model_endpoint_type: str):
             ).ask()
             model = None if len(model) == 0 else model
 
+        default_model = config.model if config.model and config.model_endpoint_type == "vllm" else ""
+
         # vllm needs huggingface model tag
         if model_endpoint_type == "vllm":
-            default_model = config.model if config.model and config.model_endpoint_type == "vllm" else ""
```

Comment:
Also should this be `config.model_endpoint_type` or `model_endpoint_type`?

Reply:
I think this is right? its using `config.model` if config is already configured for `config.model_endpoint_type == "vllm"`.